pymc3/distributions/continuous.py

```diff
@@ -2315,6 +2371,15 @@ def logcdf(value, beta, alpha):
         )
 
 
+class PyMC3GammaRV(GammaRV):
+    @classmethod
+    def rng_fn(cls, rng, alpha, beta, size):
+        return stats.gamma.rvs(alpha, scale=1 / beta, size=size, random_state=rng)
+
+
+gamma = PyMC3GammaRV()
+
+
```

Comment:
Why is this necessary?  We should be able to invert the parameter via the `Distribution.dist` interface.

We don't want the PyMC-specific interface to bleed into the `RandomVariable` interface.  `RandomVariable`s are meant to model the NumPy/SciPy interfaces (or whatever is behind their implementations), not PyMC3's interface.

Reply:
Only issue is that requires rewriting the logp and logcdf, or add the new alternative parametrizations to the distributions and have a get_x_y() for each inverted parameter. Not a lot of trouble...

Reply:
Why would you need to rewrite `logpt`/`logcdf`?

Reply:
>  or add the new alternative parametrizations to the distributions and have a get_x_y() for each inverted parameter